### ISSUE TITLE
The last returns and example section on each api docs page vanishes after reload

### ISSUE THREAD

**Issue by ST-DDT** [faker-js/faker#327]
### Describe the bug

This behaves exactly like https://github.com/faker-js/faker/issues/324 but only for the returns and example section of the last element.

````md
**Returns:** string | number

```ts
faker.address.zipCodeByState(state?: string): string | number
```
````

During reload:

![grafik](https://user-images.githubusercontent.com/1579362/151519551-131a4196-1e1b-422e-a612-45634cb55d04.png)

After a short period:

![grafik](https://user-images.githubusercontent.com/1579362/151519631-cf3077b7-9568-4ced-9b5a-f62d031ffded.png)

### Reproduction

- Enter https://fakerjs.dev/api/address.html#zip-code-by-state into your address bar and confirm

### Additional Info

We checked it and confirmed that it affects FF and Chrome, but only when serving.
You can avoid this by turning JS off or using dev mode.

**Comment by ST-DDT** [comment]
The error is not fixed yet (and is wasn't related to #330 in the first place

https://user-images.githubusercontent.com/1579362/151629455-9e7bd826-082b-4414-a06f-c536e4cb30df.mp4

)

**Comment by JessicaSachs** [comment]
Grrrrrrrr
